spec/unit/procto/to_proc_spec.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe Procto, '.to_proc' do
+  include_context 'procto'
+
+  subject { proc.call(text) }
+
+  let(:proc) { klass.to_proc }
+
+  let(:klass) do
+    Class.new do
+      include Procto.call
+
+      def initialize(text)
+        @text = text
+      end
+
+      def call
+        "Hello #{@text}"
+      end
+    end
+  end
+
+  it { should eql('Hello world') }
+
+  it 'exposes a lambda proc' do
+    expect(proc).to be_a_lambda
```

Comment:
I wonder if it wouldn't be "better", in the spirit of testing *behavior*,  to simply assert that `to_proc` returns a callable object which then behaves exactly like an explicit call to `call`? Unless I'm overlooking something, wouldn't that also further reduce the amount of test LOC by simply relying on shared context expectations? (`subject { klass.to_proc.call(text) }`). This test here feels a bit like testing an implementation detail, and by itself, it doesn't assert that an actually correct lambda is returned. I guess what I'm getting at is that this test feels a bit useless?